lib/model/classycat_schema_create.py

```diff
@@ -0,0 +1,170 @@
+import os
+import json
+import uuid
+from lib.s3 import upload_file_to_s3, file_exists_in_s3
+from lib.logger import logger
+from lib.model.model import Model
+from lib.schemas import Message, ClassyCatSchemaResponse
+
+
+class Model(Model):
+    def __init__(self):
+        super().__init__()
+        self.output_bucket = os.getenv("CLASSYCAT_OUTPUT_BUCKET")
+        self.base_prompt = (
+            "You are given a list of items in {languages} to classify. "
+            "Each item can fit into one or multiple categories. "
+            "If an item does not belong to any of the listed categories in the taxonomy, classify it as ‘Other’. "
+            "If you are an unsure about an item, classify it as ‘Unsure’. "
+            "If an item belongs to multiple categories, separate the categories with a ';'. "
+            "The input and output conform to the following format:\n"
+            "<INPUT>\n<ITEM_0>...</ITEM_0>\n<ITEM_1>...</ITEM_1>\n...\n<ITEM_N>...</ITEM_N>\n</INPUT>\n"
+            "<OUTPUT>\n<CATEGORIES_0>...</CATEGORIES_0>\n<CATEGORIES_1>...</CATEGORIES_1>\n...\n<CATEGORIES_N>...</CATEGORIES_N>\n</OUTPUT>\n\n"
+            "Here is the {schema_name} taxonomy with examples for each category included:\n{categories}\n\n"
+            "Classify the following items:\n<INPUT>\n<INSERT_ITEMS_HERE>\n</INPUT>"
+        )
+
+
+    def create_schema(self, schema_name, topics, examples, languages):
+        schema_id = str(uuid.uuid4())
+        schema_filename = f"{schema_id}.json"
+        schema_id_mapping_filename = f"{schema_name}.json"
+
+        schema = {
+            "schema_id": schema_id,
+            "schema_name": schema_name,
+            "topics": topics,
+            "examples": examples,
+            "languages": languages,
+        }
+
+        schema["prompt"] = self.generate_prompt_from_schema(schema)
+
+        upload_file_to_s3(self.output_bucket, schema_filename, json.dumps(schema))
+        # s3_client.put_object(
+        #     Bucket=output_bucket, Key=schema_filename, Body=json.dumps(schema)
+        # )
+
+        upload_file_to_s3(self.output_bucket, schema_id_mapping_filename, json.dumps({"schema_id": schema_id}))
+        # s3_client.put_object(
```

Comment:
This logs that it uploaded to S3, but is commented?  Maybe could be removed if code is using upload_file_to_s3_using_filename() instead?

Reply:
it's just old code that I have commented out and replaced with `upload_file_to_s3()`, I will remove it